shared/utils/urlHelpers.ts

```diff
@@ -49,7 +49,7 @@ export function changelogUrl(): string {
 }
 
 export function signin(service = "slack"): string {
-  return `${env.URL}/auth/${service}`;
```

Comment:
Why changed?

Reply:
this url constructor is used in exactly 1 place — the destination for the auth provider buttons. We want it to preserve the domain/subdomain the user landed on, so this url should be relative